src/components/Header.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unneeded-ternary */
+/* eslint-disable react/prop-types */
+import React from 'react';
```

Comment:
- [ ]  Kindly correct your linters instead of disabling the. they help you prevent future bugs in the app and help you apply best practices.